include/inria_wbc/controllers/controller.hpp

```diff
@@ -28,6 +28,22 @@
 #include <inria_wbc/utils/utils.hpp>
 
 namespace inria_wbc {
+
+    namespace mask
+    {
```

Comment:
What about se3_mask instead?

Reply:
I wanted something short, however se3_mask could help in case we extend these masks also to postural task or com. it can help reducing confusion.

Reply:
Yes, it's always a tradeoff. My take that 'mask' does not mean much out of context, which makes it hard to understand the calls, but on the other hand this not that complicated.

Anyway, we can keep mask if you prefer.